yt_dlp/extractor/yappy.py

```diff
@@ -3,7 +3,8 @@
     int_or_none,
     traverse_obj,
     unified_timestamp,
-    url_or_none
+    url_or_none,
+    OnDemandPagedList,
```

Comment:
move `OnDemandPagedList` to the top of utils imports
sort classes before functions, then sort alphabetically (or just use `isort`)